backend/.env.example

```
@@ -2,4 +2,8 @@ PORT=3001
 SUPABASE_URL=your_supabase_url_here
 SUPABASE_KEY=your_supabase_anon_key_here
 GEMINI_API_KEY=your_gemini_api_key_here
-FRONTEND_URL=your_front_end_url
\ No newline at end of file
+FRONTEND_URL=your_front_end_url
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The FRONTEND_URL key should go before the GEMINI_API_KEY key

(UnorderedKey)